pocs/utils/location.py

```diff
@@ -0,0 +1,75 @@
+from astroplan import Observer
+from astropy import units as u
+from astropy.coordinates import EarthLocation
+
+from pocs.utils import error
+from pocs.utils.logger import get_root_logger
+
+
+def setup_site_location_details_from_config(config, logger=None):
+    """
+     Sets up the site and location details
```

Comment:
```suggestion
     Sets up the site and location details.
```

Don't forget the periods! ;) I miss this all the time too.